### PR TITLE
4 packages from igarnier/prbnmcn-dagger at 0.0.1

### PR DESCRIPTION
**Pull request by igarnier**
This pull-request concerns:
-`prbnmcn-dagger.0.0.1`: Probabilistic programming library
-`prbnmcn-dagger-gsl.0.0.1`: Probabilistic programming library: GSL-based samplers
-`prbnmcn-dagger-stats.0.0.1`: Probabilistic programming library: prbnmcn-stats-based samplers
-`prbnmcn-dagger-test.0.0.1`: Probabilistic programming library: tests

---
* Homepage: http://github.com/igarnier/prbnmcn-dagger
* Source repo: git+https://github.com/igarnier/prbnmcn-dagger
* Bug tracker: http://github.com/igarnier/prbnmcn-dagger

---
## 0.0.1
- First release of `prbnmcn-dagger`, a library for probabilistic programming

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by mseri** [comment]
The macos CI is not reliable, don't worry about that

File: packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+opam-version: "2.0"
+synopsis: "Probabilistic programming library: GSL-based samplers"
+description: "dagger probabilistic programming library: GSL samplers"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "GPL-3.0-or-later"
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8" & >= "2.8"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam:11]:
```suggestion
  "dune" {>= "2.8"}
```
This is automatically fixed removing dune from the dependencies in the dune-project file (dune is added automatically in any case)

File: packages/prbnmcn-dagger-stats/prbnmcn-dagger-stats.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+synopsis: "Probabilistic programming library: prbnmcn-stats-based samplers"
+description:
+  "dagger probabilistic programming library: prbnmcn-stats samplers"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8" & >= "2.8"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-stats/prbnmcn-dagger-stats.0.0.1/opam:12]:
```suggestion
  "dune" {>= "2.8"}
```

File: packages/prbnmcn-dagger-stats/prbnmcn-dagger-stats.0.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "prbnmcn-dagger" {= "0.0.1"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-stats/prbnmcn-dagger-stats.0.0.1/opam:13]:
```suggestion
  "prbnmcn-dagger" {= version}
```

File: packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+opam-version: "2.0"
+synopsis: "Probabilistic programming library: tests"
+description: "dagger probabilistic programming library: tests"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8" & >= "2.8"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam:11]:
```suggestion
  "dune" {>= "2.8"}
```

File: packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  "prbnmcn-dagger" {= "0.0.1"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam:12]:
```suggestion
  "prbnmcn-dagger" {= version}
```

File: packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  "prbnmcn-stats" {>= "0.0.3"}
+  "qcheck" {>= "0.17"}
+  "prbnmcn-dagger-gsl" {= "0.0.1"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam:15]:
```suggestion
  "prbnmcn-dagger-gsl" {= version}
```

File: packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  "prbnmcn-dagger-stats" {= "0.0.1"}
```
Comment (mseri) [inline packages/prbnmcn-dagger-test/prbnmcn-dagger-test.0.0.1/opam:16]:
```suggestion
  "prbnmcn-dagger-stats" {= version}
```

File: packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  "prbnmcn-dagger" {= "0.0.1"}
```
[the remark this answers is not in the document]
Comment (mseri) [reply packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam:12]: In the dune-project file this is done by using `:version`

File: packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+synopsis: "Probabilistic programming library"
+description:
+  "A library for probabilistic programming. It takes the form of a signature for a monadic language exposing sampling and scoring primitives together with inference algorithms implementing that signature."
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8" & >= "2.8"}
```
Comment (mseri) [inline packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam:12]:
```suggestion
  "dune" {>= "2.8"}
```

File: packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam (hunk 8 above)
[the remark this answers is not in the document]
Comment (igarnier) [reply packages/prbnmcn-dagger-gsl/prbnmcn-dagger-gsl.0.0.1/opam]: Thanks a lot! I just pushed a version applying your suggestions (I fixed the dune-project so the hashes changed as well).

**Comment by mseri** [comment]
Looks good, thanks a lot.

feel free to announce the release on discuss.ocaml.org, we have a community category and an announcement tag for this purpose

**Pull request merged**